plugins/org.eclipse.osee.ats.rest/src/org/eclipse/osee/ats/rest/internal/workitem/WorkItemJsonWriter.java

```diff
@@ -487,8 +487,24 @@ private static void writeDerived(AtsApi atsApi, JsonGenerator writer, IAtsTeamWo
          writer.writeStartObject();
          writer.writeObjectField("id", art.getIdString());
          writer.writeObjectField("name", art.getName());
-         writer.writeObjectField("AtsId", teamWf.getAtsId());
+         writer.writeObjectField("AtsId",
+            atsApi.getAttributeResolver().getSoleAttributeValue(art, AtsAttributeTypes.AtsId, ""));
          writer.writeObjectField("type", art.getArtifactType().toStringWithId());
+         writer.writeObjectField("state",
+            atsApi.getAttributeResolver().getSoleAttributeValueAsString(art, AtsAttributeTypes.CurrentStateName, ""));
+         writer.writeEndObject();
+      }
+      writer.writeEndArray();
+      writer.writeArrayFieldStart("derivedto");
```

Comment:
Nit: Should this be camel case? "derivedTo"

Reply:
updated to DerivedTo to match the rest of the json